funlet-simulring/.env

```diff
@@ -1,3 +1,9 @@
+# description: The phone numbers in this comma-separated list will be called simultaneously. All phone numbers must follow the E.164 format
+# format: list(phone_number)
+# required: true
+# link: https://www.twilio.com/docs/glossary/what-e164
+FUNLET_SIMULRING_PHONE_NUMBERS="+12125551234,+17025556789"
```

Comment:
same here, the numbers being contrived

Reply:
also is simulring a customer facing term? Because this will render as the env var name, and customers may get confused

Reply:
Since Funlets are meant to be drop-in compatible with Twimlets, I'm not sure I'm able to rename this without making the Funlet more complicated. Here I think it might be best to document Simulring on this function's Wagtail page or in another visible place; this would also answer any customer questions about why the function itself is called `funlet-simulring`.